results/config/mysql-deployment.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: mysql
+spec:
+  ports:
+  - port: 3306
+  selector:
+    app: mysql
+  clusterIP: None
```

Comment:
I'm doubtful we want this setting, since I think this limits us from increasing the number of pods in the deployment.